python/python/glide/async_commands/cluster_commands.py

```diff
@@ -922,13 +922,13 @@ async def lolwut(
                 in which case the client will route the command to the nodes defined by `route`.
 
         Returns:
-            TClusterResponse[bytes]: A piece of generative computer art along with the current Redis version.
+            TClusterResponse[bytes]: A piece of generative computer art along with the current server version.
```

Comment:
Please use valkey to be consistent, some time you use Valkey and sometime server.